### DISCUSSION TITLE
H100i RGB PRO XT raising errors

### DISCUSSION THREAD

**Discussion by GabriFender** [EvanMulawski/FanControl.CorsairLink#98] [Q&A]
Hi,
I have FanControl V168 installed, with FanControl.CorsairLink v1.3.0.
I am getting the following error:
![error screenshot](https://github.com/EvanMulawski/FanControl.CorsairLink/assets/91317278/4f3f8c55-6203-44d1-a644-d5370153cfde)

The app seems to be working fine, all sensors are reading, except for the AIO pump: the pump is working fine, I can control it without issues (and I can hear it ramping up when I manually control it), but I can no longer see the pump speed, the reading is stuck at 2354 RPM no matter what I do.

This is the CorsairLink log file, no FanControl log file has been generated.
[CorsairLink.log](https://github.com/EvanMulawski/FanControl.CorsairLink/files/12512726/CorsairLink.log)

Any idea what is causing the issue?

**Comment by EvanMulawski** [comment]
Hi @GabriFender, please try [v1.4.0-beta.6](https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.4.0-beta.6)

**Reply by GabriFender** [reply]
Hi @EvanMulawski, thanks for the feedback. Tried v1.4.0-beta.6 but no luck so far, same error as before.
Here is the CorsairLink log file.
[CorsairLink.log](https://github.com/EvanMulawski/FanControl.CorsairLink/files/12514988/CorsairLink.log)

**Reply by EvanMulawski** [reply]
@GabriFender Are you using any RGB programs alongside Fan Control, e.g. OpenRGB, SignalRGB?

**Reply by GabriFender** [reply]
Yep, I am currently using the latest release of SignalRGB (v2.3.0)

**Reply by EvanMulawski** [reply]
Can you try running Fan Control without SignalRGB? Also, see #95.

**Reply by GabriFender** [reply]
Well it has been running for almost half an hour now, and no errors appeared so far. I will keep you posted in case se issue appears again, but it seems that there is a conflict with SignalRGB.
Do you know of any workarounds, other than closing SignalRGB?

Edit: grammar.

**Reply by EvanMulawski** [reply]
I do not know of a workaround - it's up to the SignalRGB developers to implement the sharing mechanism for the Platinum device family.
